### DISCUSSION TITLE
[全文搜索] algolia 搜索可以搜索到未发布的文章

### DISCUSSION THREAD

**Discussion by leewint** [tangly1024/NotionNext#1985] [Q&A]
**描述bug**
algolia 搜索可以搜索到未发布的文章。

**复现步骤**
algolia 搜索隐藏的文章的内容。

**期望的正常结果**
只能搜索到post且published的文章。

**环境**
- 操作系统:  windows
- 浏览器：edge
- 主题:nobelium

**Comment by rainnn-w** [comment]
请问这个问题解决了吗/(ㄒoㄒ)/~~

**Reply by rainnn-w** [reply]
曾经发布过后来被删除（或者设为draft）的文章也能查到，只不过点进去会一直显示LOADING...
这条record在Algolia中没有被自动删掉

**Reply by qixing-jk** [reply]
#3482 修了，才知道有这个问题，居然有的issue被close是因为被转移到了discussion
